content/docs/sidebar.json

```diff
@@ -119,6 +119,19 @@
           }
         ]
       },
+      {
+        "label": "Data Management",
+        "slug": "data-management",
+        "children": [
+          "large-dataset-optimization",
+          "defining-pipelines",
```

Comment:
The only thing that raises some concern from my end. It should not be in the data management to my mind. I think pipelines deserve their own place in the UG. It's just content should be very different as we were discussing in the PR for the pipelines.

Reply:
I struggled to keep them separate because:

1. I wasn't sure where to put `external-dependencies` (and `managing-external-data`/external outputs to a lesser extent) since they are about external data but only really make sense in the context of pipelines.
2. Pipelines in general is tightly coupled with data management now (I would like to address this on the product side, but it's not going to happen this year).
3. From a use case perspective, I can see an argument for keeping them together (a typical data engineering workflow needs data versioning and pipelines).

Still, I'm happy to keep pipelines separate and agree it seems like a more natural structure (see my comments at the bottom of #144). Where would you suggest to put `external-dependencies` and `managing-external-data`?

Reply:
> since they are about external data but only really make sense in the context of pipelines.

Hmm, it think the both could be used independently though? Especially external outputs (like versioning an external directory).

Even if that is the case, then we should move them into pipelines, out of the data management.

Reply:
Ah, I didn't catch your point about imports before. In that case, we can duplicate both sections:

```
      {
        "label": "Data Management",
        "slug": "data-management",
        "children": [
            "large-dataset-optimization",
            {
              "label": "Importing External Data",
              "source": "/docs/user-guide/pipelines/external-dependencies"
            },
            {
              "label": "Managing External Data",
              "slug": "managing-external-data"
            }
        ]
      },
      {
        "label": "Pipelines",
        "slug": "pipelines",
        "children": [
            "defining-pipelines",
            "external-dependencies",
            {
              "label": "External Outputs",
              "source": "/docs/user-guide/data-management/managing-external-data"
            }
        ]
      },
```

Reply:
yep, don't know if that is possible though :) (would be a nice feature it is possible)

Reply:
I don't think it's ideal to have page duplicates like that. I'd consider it a patch for either a product or docs content issue that should be resolved.